source/Camera/InfoUtilOther/Warmup/Warmup.csproj

```diff
@@ -0,0 +1,89 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
```

Comment:
Maybe this is a task for cleaning up the builds for everything, but perhaps we just need to get rid of the anyconfig property group in all of the C# files and that would get everything to just build on x64 only.

Reply:
We should consult SE about what is the best thing to do for all .csproj files in C# samples. But that should not belong in this PR. The .csproj file in this PR should be configured as some other samples we consider well configured, e.g. Capture sample.